### PR TITLE
feat(client): compute metrics with query filter

### LINKED ISSUE #581: Allow filtered dataset metrics.
Include basic query string filtering for dataset metrics

```python
from rubrix.metrics.token_classification import entity_labels

entity_labels(name="spacy_sm_wnut17", query="metadata.split: test").visualize()
```

### PR DESCRIPTION
**Pull request by frascuchon**
Fixes #581

### CODE REVIEW

File: src/rubrix/client/sdk/metrics/api.py [hunk 1]
```diff
@@ -47,7 +48,7 @@ def calculate_metric(
         headers=client.get_headers(),
         cookies=client.get_cookies(),
         timeout=client.get_timeout(),
-        json={},  # TODO: parameterize !!!
+        json={"query_text": query},
```
Comment (dcfidalgo) [inline src/rubrix/client/sdk/metrics/api.py:51]: does this fail if `query=None` ?

**Review by dcfidalgo — APPROVED** [review]
LGTM! Just one little question.

File: src/rubrix/client/sdk/metrics/api.py (hunk 1 above)
Comment (frascuchon) [reply src/rubrix/client/sdk/metrics/api.py:51]: API accepts these kind of request. For query=None, no filters at all will be applied

Comment (dcfidalgo) [reply src/rubrix/client/sdk/metrics/api.py:51]: :+1:

**Pull request merged**